preprocessing/raw_data/nlu-benchmark/2016-12-built-in-intents/README.md

```diff
@@ -0,0 +1,17 @@
+# Natural Language Understanding benchmark
```

Comment:
Same comment here, as with the other README.

Reply:
Alternatively, you can add a separate README under a top-level directory (maybe preprocessing/raw_data) to provide the sources for all the data.

Reply:
Thinking of the same thing! I will add a small README under the `preprocessing` folder.

Reply:
Looks good, thanks!